plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -40,10 +50,13 @@ public TestingPostgreSqlServer()
         dockerContainer = new PostgreSQLContainer<>("postgres:9.5")
                 .withDatabaseName(DATABASE)
                 .withUsername(USER)
-                .withPassword(PASSWORD);
+                .withPassword(PASSWORD)
+                .withClasspathResourceMapping("postgresql.conf", "/etc/postgresql/postgresql.conf", READ_ONLY)
```

Comment:
Does this override the builtin file? 
What configs are we losing?

Did you consider appending config changes we need into /etc/postgresql/postgresql.conf as an entry point?

Reply:
> What configs are we losing?

Nothing, other than what is set. We are still using defaults elsewhere.

Reply:
Maybe add a comment
```
// According to https://hub.docker.com/_/postgres, the configuration file needs to be based on /usr/share/postgresql/postgresql.conf.sample
// which contains just `listen_addresses = '*'`
```

also, `/etc/postgresql/postgresql.conf` -> `trino-testing-postgresql.conf` to indicate the fact the _path_ has no special meaning for the image.